source/get-started/bootable-usb.rst

```diff
@@ -33,184 +22,61 @@ Burn the |CL| image onto a USB drive
    Burning an image formats the USB drive and destroys all pre-existing
    content.  Back up your data before proceeding.
 
-#. Open a terminal window.
-
-#. Change directory to where the image resides.
-
-#. Plug in the USB drive.
-
-#. Identify the USB drive using the :command:`lsblk` command with these options:
-   ``-po NAME,SIZE,TYPE,FSTYPE,PARTLABEL,MOUNTPOINT,VENDOR,MODEL``. This shows all
-   drives attached to the system, including the primary hard disk. In the
-   example output below, there are 3 drives (`/dev/sda`, `/dev/sdb`, and `/dev/sdc`) 
-   attached, where `/dev/sda` is the primary drive and the remaining are USB drives.
-
-   .. code-block:: bash
-
-      lsblk -po NAME,SIZE,TYPE,FSTYPE,PARTLABEL,MOUNTPOINT,VENDOR,MODEL
-
-   Example output:
-
-   .. code-block:: console
-
-      NAME          SIZE VENDOR   MODEL                    TRAN   TYPE PARTLABEL                    MOUNTPOINT
-      /dev/sda    119.2G ATA      SAMSUNG_MZ7PC128HAFU-000 sata   disk                              
-      ├─/dev/sda1   450M                                          part Basic data partition         
-      ├─/dev/sda2   100M                                          part EFI system partition         
-      ├─/dev/sda3    16M                                          part Microsoft reserved partition 
-      ├─/dev/sda4  97.2G                                          part Basic data partition         
-      ├─/dev/sda5   142M                                          part EFI                          
-      ├─/dev/sda6   245M                                          part linux-swap                   [SWAP]
-      └─/dev/sda7  21.1G                                          part /                            /
-      /dev/sdb      7.5G General  UDisk                    usb    disk                              
-      └─/dev/sdb1   7.5G                                          part Microsoft Basic Data         /run/media/clear/CENA_X64FRE
-      /dev/sdc       15G          Patriot_Memory           usb    disk                              
-      └─/dev/sdc1    15G                                          part                              /run/media/clear/U
-
-   .. note::
-
-      Some Linux distros may automatically mount a USB drive when it is plugged in.
-
-#. If the USB drive you want to use is mounted, it must be umounted before
-   burning an image onto it.  Use the :command:`umount` command followed by
-   the device identifier/partition. For example, to unmount all of the 
-   ``/dev/sdc`` partitions:
+#. Launch balenaEtcher.
 
-   .. code-block:: bash
+.. figure:: /_figures/bootable-usb/balenaEtcher_Start.PNG
```

Comment:
Revise the figure like this: 

```
   .. figure:: /_figures/bootable-usb/balenaEtcher_Start.PNG       
      :scale: 100%
      :alt: BalenaEtcher Start screen
      
      Figure X: BalenaEtcher Start screen
```
Would you please revise all figures to reflect the above syntax?
If you're not comfortable with that, just let me know. I can make a final push to this PR when you complete content.

Reply:
To add to this, tab(spaces) the whole thing in under the first character of the ordered list. Otherwise it unintentionally restarts numbering of the ordered list after every figure.